### ISSUE TITLE
As reviewer, I can view private draft records to see if Biblio is missing crucial records [Review workflow case]

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#348]
# Story

As reviewer, I can view private draft records to see if Biblio is missing crucial records.

TLDR;
- Status: Private & Draft
- Actions: researcher can view new private record and send messages to researcher.

### Status
- - [ ] New, unpublished records added by researchers get the visibility status (Biblio status): Private
- - [ ] New, unpublished records added by researchers get the review status: Draft

### Workflow
- - [ ] The record does not show up in the reviewer's workflow
- - [ ] Reviewers can find new draft records added by researchers

### Visibility
- - [ ] The record is visible to involved researchers (? TBD: or only draft for editor itself?)
- - [ ] The record is invisible to the rest of the world

### Actions
- - [ ] Reviewers can view this record, and request publication and edits via message.
- - [ ] Reviewers cannot edit this record.

# Additional info

This story is part of epic #347, presented as one of the cases that fit into the review workflow that involves motivating researchers to complete records.

**Comment by mietcls** [comment]
Closed as outdated
